string sensor_name
string sensor_value
string value_type